code/game/objects/items.dm

```diff
@@ -572,3 +574,6 @@ obj/item/proc/item_action_slot_check(slot, mob/user)
 
 /obj/item/proc/is_sharp()
 	return sharpness
+
+/obj/item/proc/can_dismember()
+	return sharpness && w_class >= 3
```

Comment:
I'm going to assume that this is binary on/off flag indicating the possibility for dismemberment, like sharpness.